core/src/test/java/google/registry/testing/ReplayExtension.java

```diff
@@ -197,6 +199,7 @@ private void replayToSql() {
   }
 
   private void replayToOfy() {
+    Retrier retrier = new Retrier(new SystemSleeper(), 5);
```

Comment:
I think you want this to be an instance field on `ReplayExtension` (like the `InjectExtension`/`Clock`) rather than creating it from scratch for every single test.